internal/talent/repository.go

```diff
@@ -51,3 +53,52 @@ func (repo *talentRepository) GetTalents(ctx context.Context, pageNumber int, it
 		qm.Load(models.JobSeekerRels.Person+"."+models.PersonRels.PersonProfiles),
 		qm.OrderBy(models.JobSeekerColumns.CreatedAt+" DESC")).All(ctx, repo.executor)
 }
+
+func (repo *talentRepository) CreateTalent(ctx context.Context, params CreateTalentParams) (*models.JobSeeker, error) {
+	u1, err := uuid.NewV4()
+
+	if err != nil {
+		return nil, err
+	}
+
+	u2, err := uuid.FromString(params.PersonID)
+
+	if err != nil {
+		return nil, err
+	}
+
+	personExists, err := models.PersonExists(ctx, repo.executor, u2.String())
+
+	if err != nil && personExists == false {
+		if err.Error() == errSqlNoRows {
+			return nil, nil
+		}
+		return nil, err
+	}
+
+	var jobSeeker models.JobSeeker
+	jobSeeker.Title = params.Title
+	jobSeeker.CurrentCompany = null.StringFrom(params.CurrentCompany)
```

Comment:
Job seeker can place multiple profile with working in different companies at the same time. We can change it by introducing Hashtags like following suggestions (primary role, followed by skill, the skills will be one of those already in our continuously updating skill database)

 `dev.backend` `dev.frontend`, `dev.ios`, `qa.ios`, `devops.azure`, `devops.aws`

We can allow multiple hashtags to users so that the will be come searchable.

Reply:
@kajalsinha may I check if ID is the only thing you want me to change (as in remove) from the changes I proposed or something else too? 

Because your comment is on `CurrentCompany` ... I am bit confused.

Reply:
I think the job seeker can have Tags in database, we can discuss database model.